pkg/build/templates/modify-raw-image.sh.tpl

```diff
@@ -2,31 +2,36 @@
 set -euo pipefail
 
 #  Template Fields
-#  OutputImage   - Full path to the image to modify
-#  CombustionDir - Full path to the combustion directory
-#  ConfigureGRUB - Contains the guestfish command lines to run to manipulate GRUB configuration.
-#                  If there is no specific GRUB configuration to do, this will be an empty string.
+#  ImagePath           - Full path to the image to modify
+#  CombustionDir       - Full path to the combustion directory
+#  ConfigureGRUB       - Contains the guestfish command lines to run to manipulate GRUB configuration.
+#                        If there is no specific GRUB configuration to do, this will be an empty string.
+#  ConfigureCombustion - If true, the combustion directory will be included in the raw image
+#  RenameFilesystem    - If true, the filesystem of the image will be renamed (see below for information
+#                        on why this is needed)
 #
 # Guestfish Command Documentation: https://libguestfs.org/guestfish.1.html
 
-guestfish --format=raw --rw -a {{.OutputImage}} -i <<'EOF'
+guestfish --format=raw --rw -a {{.ImagePath}} -i <<'EOF'
   # Enables write access to the read only filesystem
   sh "btrfs property set / ro false"
 
   {{ if ne .ConfigureGRUB "" }}
-      # GRUB configuration
-      {{ .ConfigureGRUB }}
+  {{ .ConfigureGRUB }}
   {{ end }}
```

Comment:
If we drop the comment, do we even need the if statement?

Reply:
Fair, it doesn't technically need to be there. I like it for the readability; it keys in new developers to the idea that GRUB won't always be configured.